springboot/src/test/ca/jrvs/apps/trading/dao/MarketDataDaoIntTest.java

```diff
@@ -0,0 +1,51 @@
+package ca.jrvs.apps.trading.dao;
+
+import ca.jrvs.apps.trading.model.config.MarketDataConfig;
+import ca.jrvs.apps.trading.model.domain.IexQuote;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+import org.apache.http.impl.conn.PoolingHttpClientConnectionManager;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.dao.DataRetrievalFailureException;
+
+public class MarketDataDaoIntTest {
+
+  private MarketDataDao dao;
+
+  @Before
+  public void init() {
+    PoolingHttpClientConnectionManager cm = new PoolingHttpClientConnectionManager();
+    cm.setMaxTotal(50);
+    cm.setDefaultMaxPerRoute(50);
+    MarketDataConfig marketDataConfig = new MarketDataConfig();
+    marketDataConfig.setHost("https://cloud.iexapis.com/v1/");
+    marketDataConfig.setToken(System.getenv("IEX_PUB_TOKEN"));
+    dao = new MarketDataDao(cm, marketDataConfig);
```

Comment:
You can use your TestConfig object to set all of this up. I did the same thing haha

Reply:
yeah i used testconfig for the rest.